samza-sql/src/main/java/org/apache/samza/sql/data/SamzaSqlRelMessage.java

```diff
@@ -42,6 +42,14 @@
   private final List<String> names = new ArrayList<>();
   private final Object key;
 
+  public SamzaSqlRelMessage(List<String> names, List<Object> values) {
```

Comment:
nit: javadoc por public constructor

question: Looking at static method "createRelMessage(List<>, List<>)" seems to me confusing to have two methods with the same signature with different behavior on how the object is created.
      Consider making this constructor private, and converting "createRelMessage" to a constructor.

Reply:
Good feedback, The SamzaSqlRelMessage class was getting bit confusing, I did some refactoring to make it simple. Hopefully the newer version is easier to understand.